src/org/daisy/dotify/formatter/impl/row/SegmentProcessor.java

```diff
@@ -96,6 +99,9 @@
                 return "" + rs.getNumeralStyle().format(page);
             }
         };
+        this.markerRefResolver = (refs == null)
+                ? (ref) -> "??"
```

Comment:
This will probably output two question marks in the document.

1. Should you throw an exception?
2. If not, would it be an option to make this `"??"` configurable, or have a `private static final UNRESOVED_REFERENCE = "??"`

Reply:
Actually I don't think `refs` should ever be null, but this is how it is also done in `pagenumResolver` so I'm not sure.

Reply:
It seems that it is null in some tests.

Reply:
Fixed in 91869e5.